src/printer.rs

```diff
@@ -491,6 +492,14 @@ impl Printer {
         self.buffer.flush()?;
         Ok(())
     }
+
+    pub fn print_response_meta(&mut self, response_meta: ResponseMeta) -> anyhow::Result<()> {
+        let elapsed_time = response_meta.starting_time.elapsed().as_secs_f64();
```

Comment:
I don't like how `elapsed_time` is affected by when `print_response_meta()` is called.

If only there was a way to read the body without consuming `reqwest::blocking::Response`, We could have something that looked like this:

```rust
dbg!(response.metadata()); // ResponseMeta { request_time: ..., content_download_time: None }

printer.print_response_body(&mut response, response_charset, response_mime)?;

dbg!(response.metadata()); // ResponseMeta { request_time: ..., content_download_time: Some(...) }
```